pontoon/teams/static/js/request.js

```diff
@@ -74,22 +74,23 @@ var Pontoon = (function (my) {
             projects: projects,
           },
           success: function() {
-            Pontoon.endLoader('New projects request sent.', '', 5000);
+            Pontoon.endLoader('New ' + type + ' request sent.', '', 5000);
```

Comment:
This will still produce "team" instead of "language". We can just say: "New request sent".